dapps/digital-twin-detail/src/components/DigitalTwinDetail.vue

```diff
@@ -75,11 +82,15 @@
                   type="device"
                   :src="$store.state.twin.description.imgSquare"
                 />
-                <h4 class="twin-name text-center mt-2">{{ $store.state.twin.description.name }}</h4>
-                <h5 class="twin-owner text-center">{{ $store.state.twin.ownerName }}</h5>
+                <h4 class="twin-name text-center mt-2">
+                  {{ $store.state.twin.description.name }}
+                </h4>
+                <h5 class="twin-owner text-center">
+                  {{ $store.state.twin.ownerName }}
+                </h5>
                 <small
-                  class="twin-desc text-center mt-3"
-                >{{ $store.state.twin.description.description }}</small>
+                  class="twin-desc mt-3"
+                >{{ getShortDescription($store.state.twin.description.description) }}</small>
```

Comment:
what speaks against using css `ellipsis` here? Would also already handle descriptions which are shorter than 300 chars or if description is not initialized aka undefined.

Reply:
Normal ellipsis style only works with `white-space: nowrap;`, what would result in a forced one liner.  Their exists some css magic, but it's not working very well for different font-sizes.

- http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation
- https://github.com/evannetwork/ui-dapps/blob/develop/core/ui.libs/src/style/text.scss#L53

Reply:
valid point.